SConstruct

```diff
@@ -13,6 +13,12 @@ env.Append(CPPPATH=[
     "#thirdparty/openxr/include/",
     ])
 
+# Include Meta OpenXR preview headers if provided
+meta_headers = env.get("meta_headers")
+if meta_headers:
+    env.Append(CPPDEFINES=["META_HEADERS_ENABLED"])
+    env.Append(CPPPATH=[meta_headers])
```

Comment:
This is untested, but we could do something like:

```python
meta_headers_parts = os.path.split(meta_headers)
if meta_headers_parts[-1] == 'meta_openxr_preview':
    meta_headers = os.path.join(meta_headers[:-1])
```

Alternatively, we could require that the path end in "meta_openxr_preview" and then always remove it?

Or, we could require the path to the top-level "ovr_openxr_mobile_sdk_XX" directory, and then add the "/OpenXR" do that?